build/NuSpecs/MUXControlsFrameworkPackage.nuspec

```diff
@@ -40,6 +40,7 @@
     <file target="buildTransitive\Common.targets" src="MUXControls-Nuget-Common.targets"/>
     <file target="build\$ID$.props" src="MUXControls-Nuget-FrameworkPackage.props"/>
     <file target="buildTransitive\$ID$.props" src="MUXControls-Nuget-FrameworkPackage.props"/>
+    <file target="build\native\$ID$.props" src="MUXControls-Nuget-FrameworkPackage.props"/>
```

Comment:
Instead of double-including this file should we have a .props file that just includes the parent one?

Reply:
that would help make sure that anybody depending on the file existing in the old location would not be broken (though I don't expect anyone to have done that...)

Reply:
Can you say a little more about what you mean?

Reply:
Changed.